python/environment.yml

```diff
@@ -1,55 +1,54 @@
 name: housing-insights
 channels:
-- defaults
+  - defaults
 #- gramener
 dependencies:
-- libpq=9.5.4
-- mkl=11.3.3
-- openssl=1.0.2j
-- pip=8.1.2=py35_0
-- psycopg2=2.7.1=py35_0
-- python=3.5.2=0
-- python-dateutil=2.6.1=py35_0
-- pytz=2016.7=py35_0
-- setuptools=27.2.0
-- six=1.10.0=py35_0
-- sqlalchemy=1.1.11=py35_0
-- wheel=0.29.0=py35_0
-- nose=1.3.7=py35_1
+  - libpq=9.5.4
+  - mkl=11.3.3
+  - openssl=1.0.2j
+  - pip=8.1.2=py35_0
+  - psycopg2=2.7.1=py35_0
+  - python=3.5.2=0
+  - python-dateutil=2.6.1=py35_0
+  - pytz=2016.7=py35_0
+  - setuptools=27.2.0
+  - six=1.10.0=py35_0
+  - sqlalchemy=1.3.24
```

Comment:
Needed to upgrade sqlalchmey see https://github.com/sqlalchemy/sqlalchemy/issues/4463